src/dep.cc

```diff
@@ -249,13 +252,26 @@ struct RuleMerger {
         n->loc = r->loc;
     }
 
+    std::set<string> outs;
```

Comment:
nit: it appears that the local custom is to not have the `std::` prefix

Reply:
This is unused, from testing. Removed.